config/jobs/kubernetes-sigs/secrets-store-csi-driver/secrets-store-csi-driver-config.yaml

```diff
@@ -167,6 +167,39 @@ presubmits:
       testgrid-tab-name: pr-secrets-store-csi-driver-e2e-azure
       description: "Run e2e test with azure provider for Secrets Store CSI driver."
       testgrid-num-columns-recent: '30'
+  - name: pull-secrets-store-csi-driver-e2e-deploy-manifest-azure
+    decorate: true
+    decoration_config:
+      timeout: 25m
+    always_run: true
```

Comment:
Let's also add `optional: true` for now. Based on how stable the tests are, we can make it `required` in a follow up PR.

Reply:
done